binary-array-ld-cli/src/main/kotlin/net/bald/BinaryArrayConvertCli.kt

```diff
@@ -34,14 +38,23 @@ class BinaryArrayConvertCli {
 
     private fun doRun(opts: CommandLineOptions) {
         val inputLoc = opts.inputLoc ?: throw IllegalArgumentException("First argument is required: NetCDF file to convert.")
-        val ba = NetCdfBinaryArray.create(inputLoc, opts.uri)
-        val model = ba.use(ModelBinaryArrayConverter::convert)
+        val ba = NetCdfBinaryArray.create(inputLoc, opts.uri).withContext(opts.contextLocs)
+        val model = ba.use {
```

Comment:
Another Kotlin question:
Are lines 42 and 43 functionally any different from how the val model was created before?

Reply:
No, this change is accidental. Personally I think the old way is neater but I don't feel strongly about it.

Reply:
If you prefer the previous way, then probably best to change it back. I think it makes it more consistent with the rest of the code written how it was before. 
Thanks!

Reply:
Done.

Reply:
Thanks!